### PR TITLE
Add application name to the logger

### PR DESCRIPTION
**Pull request by ly-william**
A common idiom is to specify the app name in the logstash arguments. That adds a "type" field to the created JSON.

### CODE REVIEW

**Comment by ripcurld00d** [comment]
Hi @ly-william,

Thanks for your PR!!! :confetti_ball: I appreciate the time you took to contribute.
I will be reviewing your commit in the next days as I am too busy right now.

:grinning:

File: logstash.go [hunk 1]
```diff
@@ -10,20 +10,21 @@ import (
 // Hook represents a connection to a Logstash instance
 type Hook struct {
 	conn net.Conn
+	appName string
 }
 
 // NewHook creates a new hook to a Logstash instance, which listens on
 // `protocol`://`address`.
-func NewHook(protocol, address string) (*Hook, error) {
+func NewHook(protocol, address string, appName string) (*Hook, error) {
```
Comment (ripcurld00d) [inline logstash.go]: a small nitpick: change `NewHook(protocol, address **string**, appName string)` to `NewHook(protocol, address, appName string)`

File: logstash.go [hunk 2]
```diff
@@ -10,20 +10,21 @@ import (
 // Hook represents a connection to a Logstash instance
 type Hook struct {
 	conn net.Conn
+	appName string
```
Comment (ripcurld00d) [inline logstash.go]: can you align the struct so it will look like:

```
type Hook struct {
    conn    net.Conn
    appName string
}
```

**Comment by ly-william** [comment]
Thanks for your feedbacks, I updated the PR accordingly :)

**Comment by ripcurld00d** [comment]
LGTM! (tested on my laptop)
@ly-william thanks a lot!

**Pull request merged**